doc/man/8/ceph.rst

```diff
@@ -1024,6 +1033,16 @@ Usage::
 
 	ceph osd setmaxosd <int[0-]>
 
+Subcommand ``set-require-min-compat-client`` configures the oldest Ceph client
+version allowed to connect to the cluster. Once set, Ceph will prevent you from
+enabling features that will break compatibility with those clients. Please note,
+This subcommand requires that all connected daemons and clients support the
+features offered by the given <version>.
+
+Usage::
+
+    ceph osd set-require-min-compat-client <version> {--yes-i-really-mean-it}
```

Comment:
I would leave off the force argument; the CLI will prompt the user for it only if there are older connected clients.